### PR TITLE
🛢 Add consistent URL ("/latest/") for most recent product version.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- Navigate around site; see `/latest/` in URL

### CODE REVIEW

File: metadeploy/consumers.py [hunk 1]
```diff
@@ -113,7 +113,8 @@ async def receive_json(self, content, **kwargs):
         scratch_org_id = self.scope["session"].get("scratch_org_id", None)
         if uuid and uuid != scratch_org_id:
             self.scope["session"]["scratch_org_id"] = uuid
-            self.scope["session"].save()
+            # TODO: Without this, I was getting an error after creating a scratch org:
```
Comment (jgerigmeyer) [inline metadeploy/consumers.py:116]:
```
HTTP POST /api/plans/NzN9R4X/scratch_org/ 202 [0.07, 127.0.0.1:32906]
Exception inside application: You cannot call this from an async context - use a thread or sync_to_async.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/channels/staticfiles.py", line 44, in __call__
    return await self.application(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/routing.py", line 71, in __call__
    return await application(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/sessions.py", line 47, in __call__
    return await self.inner(dict(scope, cookies=cookies), receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/sessions.py", line 263, in __call__
    return await self.inner(wrapper.scope, receive, wrapper.send)
  File "/usr/local/lib/python3.8/site-packages/channels/sessions.py", line 47, in __call__
    return await self.inner(dict(scope, cookies=cookies), receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/sessions.py", line 263, in __call__
    return await self.inner(wrapper.scope, receive, wrapper.send)
  File "/usr/local/lib/python3.8/site-packages/channels/auth.py", line 185, in __call__
    return await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/middleware.py", line 26, in __call__
    return await self.inner(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/routing.py", line 150, in __call__
    return await application(
  File "/usr/local/lib/python3.8/site-packages/channels/consumer.py", line 94, in app
    return await consumer(scope, receive, send)
  File "/usr/local/lib/python3.8/site-packages/channels/consumer.py", line 58, in __call__
    await await_many_dispatch(
  File "/usr/local/lib/python3.8/site-packages/channels/utils.py", line 51, in await_many_dispatch
    await dispatch(result)
  File "/usr/local/lib/python3.8/site-packages/channels/consumer.py", line 73, in dispatch
    await handler(message)
  File "/usr/local/lib/python3.8/site-packages/channels/generic/websocket.py", line 194, in websocket_receive
    await self.receive(text_data=message["text"])
  File "/usr/local/lib/python3.8/site-packages/channels/generic/websocket.py", line 257, in receive
    await self.receive_json(await self.decode_json(text_data), **kwargs)
  File "/app/metadeploy/consumers.py", line 116, in receive_json
    self.scope["session"].save()
  File "/usr/local/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 81, in save
    return self.create()
  File "/usr/local/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 51, in create
    self._session_key = self._get_new_session_key()
  File "/usr/local/lib/python3.8/site-packages/django/contrib/sessions/backends/base.py", line 196, in _get_new_session_key
    if not self.exists(session_key):
  File "/usr/local/lib/python3.8/site-packages/django/contrib/sessions/backends/db.py", line 47, in exists
    return self.model.objects.filter(session_key=session_key).exists()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 809, in exists
    return self.query.has_results(using=self.db)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/query.py", line 537, in has_results
    return compiler.has_results()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/compiler.py", line 1126, in has_results
    return bool(self.execute_sql(SINGLE))
  File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/compiler.py", line 1154, in execute_sql
    cursor = self.connection.cursor()
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 24, in inner
    raise SynchronousOnlyOperation(message)
django.core.exceptions.SynchronousOnlyOperation: You cannot call this from an async context - use a thread or sync_to_async.
```

**Comment by jgerigmeyer** [comment]
[I want the URL for the most recent version of a plan to be a permalink.](https://trello.com/c/nZGUUMO8/192-i-want-the-url-for-the-most-recent-version-of-a-plan-to-be-a-permalink)

File: metadeploy/consumers.py (hunk 1 above)
Comment (jerivas) [reply metadeploy/consumers.py:116]: From the error message makes sense 👍

**Review by jerivas — APPROVED** [review]
LGTM 🚀 A small observation (and this might already been resolved in Slack/Trello): I couldn't find a clear way of getting a link to the specific version in case I don't want to point to the latest one. Not sure if that's a valid use case for MetaDeploy 🤷‍♂️

**Comment by jgerigmeyer** [comment]
> I couldn't find a clear way of getting a link to the specific version in case I don't want to point to the latest one. Not sure if that's a valid use case for MetaDeploy 🤷‍♂️

That's a good point. I generally think we're trying to avoid that -- @davisagli what do you think?

**Comment by davisagli** [comment]
Yeah that's right, we want to discourage that because generally using an old installer at some point in the future is only a valid use case for internal folks trying to reproduce some issue -- we push upgrade customers to the latest version and want to make sure they are starting out there as well.

**Comment by jgerigmeyer** [comment]
@davisagli I think this is ready to go. Would you prefer to review it here, or should I open up a PR upstream into https://github.com/SFDO-Tooling/MetaDeploy?

File: src/js/store/org/actions.ts [hunk 2]
```diff
@@ -32,10 +32,13 @@ export const fetchOrgJobs =
       );
       if (response && window.socket) {
         for (const orgId of Object.keys(response)) {
-          window.socket.subscribe({
-            model: 'org',
-            id: orgId,
-          });
+          /* istanbul ignore else */
+          if (orgId !== 'null') {
```
Comment (davisagli) [inline src/js/store/org/actions.ts:36]: I don't want to know why it would be the string 'null', do I

**Review by davisagli — APPROVED** [review]

**Pull request merged**